src/vm/eventpipe.cpp

```diff
@@ -107,7 +107,7 @@ void EventPipe::Shutdown()
 
 void EventPipe::Enable(
     LPCWSTR strOutputPath,
-    uint circularBufferSizeInMB,
+    unsigned int circularBufferSizeInMB,
```

Comment:
The chain is 
`eventpipe.cpp -> common.h -> ceeload.h -> (IF FEATURE_READYTORUN -> readytoruninfo.h -> nativeformatreader.h -> uint)`
Of course, I can still separate, but it seems to me that this is a direct binding.